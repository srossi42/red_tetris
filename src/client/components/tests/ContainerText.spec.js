import React from 'react'
import { shallow } from 'enzyme';
import ContainerText from '../ContainerText';
import toJson from "enzyme-to-json"


describe('<ContainerText/> Component', () => {
  it('rendering correctly ContainerText', () => {
    let wrapper = shallow(
      <ContainerText text="lol"/>
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
