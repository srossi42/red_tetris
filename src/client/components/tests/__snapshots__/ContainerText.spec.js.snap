// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContainerText/> Component rendering correctly ContainerText 1`] = `
<div
  style={
    Object {
      "alignContent": "center",
      "alignItems": "center",
      "backgroundColor": "black",
      "border": "2px solid black",
      "display": "flex",
      "height": "80vh",
      "justifyContent": "center",
      "width": "40vh",
    }
  }
>
  <span
    style={
      Object {
        "color": "white",
        "margin": "20px",
        "textAlign": "center",
      }
    }
  >
    lol
  </span>
</div>
`;
